charts/console/values.go

```diff
@@ -58,7 +58,7 @@ type Values struct {
 	InitContainers               InitContainers                    `json:"initContainers"`
 	SecretMounts                 []SecretMount                     `json:"secretMounts"`
 	Secret                       SecretConfig                      `json:"secret"`
-	Enterprise                   Enterprise                        `json:"enterprise"`
+	LicenseSecretRef             SecretKeyRef                      `json:"licenseSecretRef"`
```

Comment:
If it's not too much extra work can we turn this into a `*corev1.SecretKeySelector` so it's using a more standard type?

I would also recommend turning the default value of this in values.yaml to `null` and then providing an example object in the docs instead of specifying all the keys as being empty strings.

Reply:
Changed to be `*corev1.SecretKeySelector`